CoEpi/domain/model/CENKey.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import CryptoKit
+import RealmSwift
+import Security
+
+struct CENKey : Codable {
+    var cenKey: String?
```

Comment:
This shouldn't be optional. If there's not key then this struct doesn't make sense.